freq/settings.py

```diff
@@ -193,7 +191,7 @@
 #     os.path.join(BUILDOUT_DIR, 'bower_components'),
 #     # ^^^ bower-managed files.
 # ]
-
+SESSION_SAVE_EVERY_REQUEST = True
```

Comment:
Is dit echt nodig? Anders misschien commentaarregel erboven/eronder zetten met kort de reden.